tests/Routing/RoutingRouteTest.php

```diff
@@ -549,18 +549,21 @@ public function testModelBindingWithBindingClosure()
     public function testGroupMerging()
     {
         $old = ['prefix' => 'foo/bar/'];
-        $this->assertEquals(['prefix' => 'foo/bar/baz', 'namespace' => null, 'where' => []], Router::mergeGroup(['prefix' => 'baz'], $old));
+        $this->assertEquals(['prefix' => 'foo/bar/baz', 'namespace' => null, 'as' => null, 'where' => []], Router::mergeGroup(['prefix' => 'baz'], $old));
```

Comment:
I changed these tests to keep everything consistent. The `mergeGroup` method of Router "forced" `null` values for `prefix` and `namespace` when they weren't set manually. Everything else about the `as` option is consistent with them, and so I changed its structure in `mergeGroup` to match those two. Making it always show up (like they do) means that it gets a null when it hasn't been set at all, which means that calling `::mergeGroup()` will create a null value for `as`.

I added the `null`s for `as` just like the existing `null`s for `prefix` and `namespace`.